deployments/jwebbinar/config/dev.yaml

```diff
@@ -3,3 +3,13 @@ jupyterhub:
     continuous:
       # Don't need continuous pre-puller on staging
       enabled: true
+  singleuser:
```

Comment:
@jmatuskey - these should no longer be here, right?  Weren't they moved somewhere more general?